### ISSUE TITLE
Pyblish-pype: Toggle instance family

### ISSUE THREAD

**Issue by tokejepsen** [pypeclub/pype#300]
Would be great to be able to toggle a whole family of instances.

**Comment by iLLiCiTiT** [comment]
I think this can be done with checkbox or right click menu on family group item.
- checkbox is visible but dangerous because can be accidentally clicked
- right click is not visible but is safe(r)
This won't work in Artist view...

Any other UI idea how to achieve that?

**Comment by tokejepsen** [comment]
When I did the hide section for pyblish-qml we iterated through various versions of toggling the family group; https://github.com/pyblish/pyblish-qml/pull/275

I would say that checkbox is not needed, cause it might lead to confusion. For example someone unchecks all instances manually, then goes to try and uncheck the family group.
I think pyblish-qml way of clicking on the family groups name seems to work well.
